via/views/exceptions.py

```diff
@@ -124,7 +124,7 @@ def _serialise_requests_info(request, response):
         try:
             data["text"] = response.text[:5000]
 
-        except Exception:  # pylint: disable=broad-except
+        except Exception:  # pylint: disable=broad-except  # noqa: BLE001
```

Comment:
FIXME: need to grep for `pylint` comments and remove them, there's still a bunch in Via.